cli/helm/values.go

```diff
@@ -411,7 +411,7 @@ type TransparentProxy struct {
 }
 
 type Metrics struct {
-	DefaultEnabled              string `yaml:"defaultEnabled"`
+	DefaultEnabled              bool   `yaml:"defaultEnabled"`
```

Comment:
This appears to have been an oversight for a flag that should actually be a `bool`, can revert if out of scope but felt pertinent to fix for parity with similar proxy lifecycle management config.